src/scripts/main.js

```diff
@@ -1,3 +1,30 @@
 'use strict';
 
-// write code here
+const employees = document.querySelectorAll('li');
+
+const sortList = function(list) {
+  const copyOfList = Array.from(list);
```

Comment:
```suggestion
  const copyOfList = [...list];
```
You can also get copy like this